tasks/handle_container_conf_d.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Ensure containers.d exists - system
+  file:
+    path: "{{ __podman_containers_conf_system|dirname }}"
+    state: "directory"
```

Comment:
ansible-lint says you must have `owner`, `group`, and `mode` for files/directories created by the role.